lib/src/layer/polyline_layer.dart

```diff
@@ -150,12 +150,15 @@ class PolylinePainter extends CustomPainter {
           : paint.color = polylineOpt.color;
     }
 
-    final filterPaint = Paint()
-      ..color = polylineOpt.borderColor.withAlpha(255)
-      ..strokeWidth = polylineOpt.strokeWidth
-      ..strokeCap = StrokeCap.round
-      ..strokeJoin = StrokeJoin.round
-      ..blendMode = BlendMode.dstOut;
+    Paint filterPaint;
+    if (polylineOpt.borderColor != null) {
```

Comment:
Is this change intentional?

Reply:
yeah, that's from an much older commit. I recall that I got null reference exceptions when borderColor wasn't set

Reply:
Do you see any problem with that?

Reply:
No problem, just harder to review :)